jekyll.py

```diff
@@ -301,6 +302,7 @@ def callback(self, index):
             f = self.posts[index][1]
             syntax = self.get_syntax(self.posts[index][0])
             output_view = self.window.open_file(f)
+            output_view.settings().set("is_jekyll_post", True)
```

Comment:
One issue with this implementation method is that the `jekyll_list_uploads` command is only available for posts that were created using `jekyll_new_post` or `jekyll_new_draft` **AND** exist in the current view. This is because you are using view based settings. This seems too limiting, as I cannot use the feature if I open a post I created earlier and want to add an upload to it.

I'm also wondering if you want to subclass `JekyllListPostsBase` instead of re-writing the logic for searching and listing files. It seems to better align with the [DRY][dry] methodoloy of Python.

[dry]: http://en.wikipedia.org/wiki/Don%27t_repeat_yourself